### ISSUE TITLE
Consider: Make Dropdown Accessible

### ISSUE THREAD

**Issue by jpdevries** [Thinkful/ui#314]
The current div based implementation of the `<Dropdown>` is not accessible to keyboards and screen readers. It also does not support native features like "autocomplete" that allow you to type the first few characters of an option to select it.
